specification/dfp/resource-manager/Microsoft.DFP/preview/2021-02-01-preview/dfp.json

```diff
@@ -0,0 +1,782 @@
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Dfp/instances/{instanceName}": {
+      "get": {
+        "description": "Gets details about the specified instances.",
+        "x-ms-examples": {
+          "Get details of an instance": {
+            "$ref": "./examples/getInstance.json"
+          }
+        },
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_GetDetails",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "instanceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the instance. It must be a minimum of 3 characters, and a maximum of 63."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/DFPInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "description": "Provisions the specified DFP instance based on the configuration specified in the request.",
+        "x-ms-examples": {
+          "Create instance": {
+            "$ref": "./examples/createInstance.json"
+          }
+        },
+        "tags": [
+          "Instances"
+        ],
+        "operationId": "Instances_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "instanceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63,
+            "description": "The name of the DFP instances. It must be a minimum of 3 characters, and a maximum of 63."
+          },
+          {
+            "name": "instanceParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DFPInstance"
+            },
+            "description": "Contains the information used to provision the DFP instance."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
so the put is not going to be a long-running/async operation?

Reply:
Added this option